test/module-loader/module-loader.test.ts

```diff
@@ -108,15 +149,10 @@ describe('ModuleLoader:', () => {
     for (const module of data.modules) {
       it(`${module.path}`, async () => {
         const config = new Config({root: process.cwd()})
-        const configESM = new Config({root: process.cwd()})
-
         await config.load()
-        await configESM.load()
 
-        // Must set type manually for ESM config.
-        configESM.pjson.type = 'module'
-
-        const result = await ModuleLoader.loadWithData(module.isESM ? configESM : config, module.path)
+        // const result = await ModuleLoader.loadWithData(module.isESM ? configESM : config, module.path)
```

Comment:
Is this needed?

Reply:
Nope.. Left over from being able to remove creating a special config for ESM testing. The ModuleLoader update / fix is a bit more flexible now